pkg/image/archive.go

```diff
@@ -39,6 +40,7 @@ func (img *Archive) InstantiateManifest(m *manifest.Manifest,
 	osPipeline.OSVersion = img.OSVersion
 
 	tarPipeline := manifest.NewTar(buildPipeline, osPipeline, "archive")
+	tarPipeline.NumericOwner = common.ToPtr(true)
```

Comment:
This is totally not Go, this should have been negative non-pointer. Something like: `SkipNumericOwner bool`